impl/src/bin/cargo-raze.rs

```diff
@@ -54,8 +50,8 @@ Generate BUILD files for your pre-vendored Cargo dependencies.
 
 Usage:
     cargo raze (-h | --help)
-    cargo raze [--verbose] [--quiet] [--color=<WHEN>] [--dryrun] [--cargo-bin-path=<PATH>] [--deprecated-use-cargo-internals]
-    cargo raze <buildprefix> [--verbose] [--quiet] [--color=<WHEN>] [--dryrun] [--cargo-bin-path=<PATH>] [--deprecated-use-cargo-internals]
+    cargo raze [--verbose] [--quiet] [--color=<WHEN>] [--dryrun] [--cargo-bin-path=<PATH>]
```

Comment:
Ideally the deletion of the old internals mode can be decoupled from the update of the cargo-metadata mode from being shell command based to using the dedicated crate.

(I'm going to assume this is implied in the rest of the code -- I won't comment on things that are part of the old mode and are being deleted here)

Is it possible to do that in this change? If not, I'll have to go make sure the feature gaps described in https://github.com/google/cargo-raze/pull/149#issuecomment-621493706 before I'm comfortable with this change being submitted.

Reply:
I'm going to try to land #149 first - IMO, it simplifies the assumptions of this code - but please feel free to push back on me if you think that's the wrong direction!